extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1039,9 +1039,21 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @private
    */
   handleKeyboardEvents_(event) {
-    const code = event.keyCode;
-    if (code == KeyCodes.ESCAPE) {
-      this.close_();
+    if (this.active_) {
```

Comment:
1. Make this a flat short circuit clause set. Easier to debug and reason about.
2. nit: `isActive_` (probably from a previous PR, feel free to leave as is).
3. `getImpl` is safe but `implementation_` isn't.
4. Prevent loosening types. I believe the following snippet fixes the `@type {?}` issue.
5. The same attributes are being passed to `goCallback`, you should hoist to a separate method.
6. Mini-nit: `onKeyDown_` to reflect the actual event.
7. Use ES2015 deconstructors where it makes sense.

```javascript
/** ... */
onKeyDown_(e) {
  if (!this.active_) {
    return;
  }
  const {keyCode} = e;
  if (keyCode == KeyCodes.ESCAPE) {
    this.close_();
    return;
  }
  if (keyCode != Keycodes.LEFT_ARROW &&
      keyCode != Keycodes.RIGHT_ARROW) {
    return;
  }
  const direction = keyCode == Keycodes.RIGHT_ARROW ? 1 : -1;
  this.maybeSlideCarousel_(direction);
  return;  
}

/**
 * @param {number} direction 1 or -1.
 * @private
 */
maybeSlideCarousel_(direction) {
  const isGalleryView = this.container_.hasAttribute('gallery-view');
  if (isGalleryView) {
    return;
  }
  dev().assert(this.carousel_).getImpl().then(carousel => {
    carousel.goCallback(direction, /* animate */ true, /* autoplay */ false);
  });
}
```